Source/DCSFlightpanels/PanelUserControls/PreProgrammed/Cdu737UserControlA10C.xaml.cs

```diff
@@ -0,0 +1,137 @@
+﻿
+namespace DCSFlightpanels.PanelUserControls.PreProgrammed
+{
+
+    using System.Windows;
+    using System.Windows.Controls;
+
+
+    using NonVisuals;
+    using NonVisuals.EventArgs;
+    using NonVisuals.Interfaces;
+
+    using NonVisuals.CockpitMaster.Preprogrammed;
+    using DCSFlightpanels.Interfaces;
+    using System;
+    using NonVisuals.Saitek.Panels;
+    using NonVisuals.Saitek.Switches;
+
+    /// <summary>
+    /// Logique d'interaction pour Cdu737UserControlA10C.xaml
+    /// </summary>
+    /// 
+    public partial class Cdu737UserControlA10C : UserControlBase,
+        IGamingPanelListener,
+        IGamingPanelUserControl
+    {
+        private readonly CDU737PanelA10C _CDU737PanelA10C;
+
+        public Cdu737UserControlA10C(HIDSkeleton hidSkeleton, TabItem parentTabItem)
+        {
+            InitializeComponent();
+            ParentTabItem = parentTabItem;
+
+            _CDU737PanelA10C = new CDU737PanelA10C(hidSkeleton);
+            //_HIDSkeleton = hidSkeleton;
+            AppEventHandler.AttachGamingPanelListener(this);
+
+            HideAllImages();
+
+        }
+        private bool _disposed;
+        // Protected implementation of Dispose pattern.
+        protected override void Dispose(bool disposing)
+        {
+            if (!_disposed)
+            {
+                if (disposing)
+                {
+                    _CDU737PanelA10C.Dispose();
+                    AppEventHandler.DetachGamingPanelListener(this);
+
+                }
+
+                _disposed = true;
+            }
+
+            // Call base class implementation.
+            base.Dispose(disposing);
+        }
+
+        private bool _once = true;
+
+
+        private void CDU737UserControl_OnLoaded(object sender, RoutedEventArgs e)
+        {
+            UserControlLoaded = true;
+
```

Comment:
Not so important but I prefer avoiding uneccessary blank lines, makes the code more compact and easier to read.
I tend to space functions with 1 blank line. And inside function, add only blank line between block of code of the same topic or to make a clear distinctinction.

But I understand that everyone have their preferences ;-)